numpy/testing/tests/test_warnings.py

```diff
@@ -0,0 +1,86 @@
+"""
+Tests which scan for certain occurances in the code, they may not find
+all of these occurances but should catch almost all.
+"""
+
+
+from __future__ import division, absolute_import, print_function
+
+
+import sys
+if sys.version_info >= (3, 4):
```

Comment:
Not sure if the tokenize is actually necessary to be honest, I stole it from a scipy test (though that one did regex and not AST matching).